doc/user/content/manage/user-management/rbac-newrole.md

```diff
@@ -0,0 +1,220 @@
+---
+title: "Create a New User and Role"
+description: ""
+menu:
+  main:
+    parent: user-management
+    weight: 16
+---
+
+This guide walks you through creating a new user and new role in Materialize. By
+the end of this tutorial you will:
+
+* Invite a new user in the Materialize Console
+* Create a new role in your Materialize account using `psql`
+* Apply privileges to the new role
+* Assign the new role to the new user
+
+## Before you begin
+
+In this guide, you'll use `psql` to manage users in Materialize, so be sure you
+have `psql` installed locally.
+
+If you have not signed up for a free trial, [sign up now](https://materialize.com/register/?utm_campaign=General&utm_source=documentation).
+
+
+## Invite a new user
+
+First, login to your Materialize console and navigate to Account > Account
+Settings > Users.
+
+Click **Invite User** and fill in the user information.
+
+The **Organization Admin** and **Organization Member** roles refer to console
+privileges and are not related to RBAC roles.
```

Comment:
**Organization Admin** will cause the role in Materialize to be created with superuser privileges.